### ISSUE TITLE
Still some encoding issues

### ISSUE THREAD

**Issue by sylvinus** [commonsearch/cosr-back#9]
There seem to be a few cases left where we get � characters in search results:

- https://uidemo.commonsearch.org/?g=en&q=sket

Check to see if they can be fixed, and prefer ignoring the error completely than showing any �.

**Comment by glebourgeois** [comment]
I did not look yet into these two examples specifically, but considering this : 
```
if self.doc.source_headers.get("content-type"):
            header_encoding = get_encoding_from_content_type(self.doc.source_headers["content-type"])
            if header_encoding:
                return header_encoding
```

I can assure you that you cannot trust the encoding specified in headers ; they are quite often badly declared.

**Comment by sylvinus** [comment]
Indeed! 

Do you have another solution in mind? Would be great to avoid doing chardet on all documents just to fix the <1% cases where they are badly declared :(

**Comment by glebourgeois** [comment]
Nope, you just cannot trust your input data, that's the beauty of the web ;)
And as you know, garbage in, garbage out.
